test/runtests.jl

```diff
@@ -98,20 +110,20 @@ end
     @test_throws AssertionError("MultiLevelPoisson requires size=a2ⁿ, where n>2") Poisson_setup(MultiLevelPoisson,(15+2,3^4+2))
 
     err,pois = Poisson_setup(MultiLevelPoisson,(10,10))
-    @test pois.levels[3].D == Float32[0 0 0 0; 0 -2 -2 0; 0 -2 -2 0; 0 0 0 0]
+    @test pois.levels[3].D == Float32[0 0 0 0; 0 -2 -3 0; 0 -3 -4 0; 0 0 0 0]
```

Comment:
Why change this?

Reply:
This is actually a bad error, I didn't realise when I did that, but it's related the the BC on the \mu_0 field in the levels below 1 of the multigrid.